daprdocs/content/en/reference/components-reference/supported-configuration-stores/azure-appconfig-configuration-store.md

```diff
@@ -0,0 +1,86 @@
+---
+type: docs
+title: "Azure App Configuration"
+linkTitle: "Azure App Configuration"
+description: Detailed information on the Azure App Configuration configuration store component
+aliases:
+  - "/operations/components/setup-configuration-store/supported-configuration-stores/setup-azure-appconfig/"
+---
+
+## Component format
+
+To setup Azure App Configuration configuration store create a component of type `configuration.azure.appconfig`.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: configuration.azure.appconfig
+  version: v1
+  metadata:
+  - name: host
+    value: <HOST>
+  - name: connectionString
+    value: <CONNECTIONSTRING>
+  - name: maxRetries
+    value: # Optional
+  - name: retryDelay
+    value: # Optional
+  - name: maxRetryDelay
+    value: # Optional
+  - name: azureEnvironment # Optional, defaults to AZUREPUBLICCLOUD
+    value: "AZUREPUBLICCLOUD"
+  # See authentication section below for all options
+  - name: azureTenantId # Optional
+    value: "[your_service_principal_tenant_id]"
+  - name: azureClientId # Optional
+    value: "[your_service_principal_app_id]"
+  - name: azureCertificateFile # Optional
+    value : "[pfx_certificate_file_fully_qualified_local_path]"
+
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field             | Required | Details | Example |
+|-------------------|:--------:|---------|---------|
+| host              | N        | Endpoint for the Azure App Configuration instance. No Default  | `https://dapr.azconfig.io`
+| connectionString  | N        | Connection String the Azure App Configuration instance. No Default. Can be `secretKeyRef` to use a secret reference  | `Endpoint=https://foo.azconfig.io;Id=osOX-l9-s0:sig;Secret=00000000000000000000000000000000000000000000`
+| maxRetries        | N         | Maximum number of retries before giving up. Defaults to `3` | `5`, `10`
+| retryDelay        | N         | RetryDelay specifies the initial amount of delay to use before retrying an operation. The delay increases exponentially with each retry up to the maximum specified by MaxRetryDelay. Defaults to `4` seconds; `"-1"` disables delay between retries. | `4000000000`
+| maxRetryDelay     | N         | MaxRetryDelay specifies the maximum delay allowed before retrying an operation. Typically the value is greater than or equal to the value specified in RetryDelay. Defaults to `120` seconds; `"-1"` disables the limit | `120000000000`
+
+**Note**: either `host` or `connectionString` must be specified.
```

Comment:
If they **must** be specified, should we put "Y" in their required columns?

Reply:
It's one or the other. I think that adding "Y" to both fields can be misleading.